### PR TITLE
Add test for #71.

### PR DESCRIPTION
**Pull request by satazor**

### CODE REVIEW

File: src/translate.spec.js [hunk 1]
```diff
@@ -60,4 +63,46 @@ describe('translate enhancer', () => {
             expect(TranslatedComponent.displayName).toBe(translatedDefaultName);
         });
     });
+
+    it('should not re-render component on every non-related dispatch call', async () => {
+        let pChanged = false;
+        let dispatched = true;
```
Comment (satazor) [inline src/translate.spec.js:69]: This should be changed to false by default..

**Comment by satazor** [comment]
cc @guillaumearm

File: src/translate.spec.js (hunk 1 above)
Comment (guillaumearm) [reply src/translate.spec.js:69]: Sure, but I have some doubt about the utility of this boolean for this test, what do you think ?

Comment (satazor) [reply src/translate.spec.js:69]: I just added it to be sure that the action was dispatched. It ensures that if the react internals introduce any delay in life cycle methods, the test will fail.

**Pull request merged**